css/homepage.scss

```diff
@@ -31,6 +31,67 @@
  }
 }
 
+#description {
+  margin-top: 5rem;
+  margin-bottom: 5rem;
+  .icon {
+    line-height: 1;
+    vertical-align: middle;
+    font-size: 20em;
```

Comment:
I specified the size with the unit used by semantic-ui. Don't you think it's better to use the same metric ?